downstream/assemblies/platform/assembly-disconnected-installation.adoc

```diff
@@ -13,11 +13,21 @@ If you are not connected to the internet or do not have access to online reposit
 
 Before installing {PlatformNameShort} on a disconnected network, you must meet the following prerequisites:
 
+<<<<<<< HEAD
```

Comment:
When I do a build of the doc it is showing all these merge conflict lines messages.